lib/web/fetch/util.js

```diff
@@ -1428,6 +1428,34 @@ function getDecodeSplit (name, list) {
   return gettingDecodingSplitting(value)
 }
 
+const textDecoder = new TextDecoder()
+
+/**
+ * @see https://encoding.spec.whatwg.org/#utf-8-decode
+ * @param {Buffer} buffer
+ */
+function utf8DecodeBytes (buffer) {
+  if (buffer.length === 0) {
```

Comment:
```suggestion
  // fast-path
  if (buffer.length === 0) {
```

Reply:
this was only moved from body.js to prevent a circular import, but I think it's rather obvious what the point of it is